src/BaGetter/Startup.cs

```diff
@@ -89,5 +91,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             baget.MapEndpoints(endpoints);
         });
+
+        app.UseHealthChecks("/healthz");
```

Comment:
```suggestion
        app.UseHealthChecks("/health");
```

Reply:
I recently learning of this very nice github feature to request specific changes and directly commit it with a click. Please use a common name for the health check endpoint, as this is complicated enough to find without an OpenAPI endpoint.

Reply:
I would more like an overwriteable endpoint, but `health` as default.

Reply:
I will change to be configurable with a default of /health, as @seriouz suggested.
The name healthz is very common in containerized applications, which is why I am used to it.
[See this stackoverflow question](https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f)